src/java/apiview-java-processor/src/main/java/com/azure/tools/apiview/processor/analysers/util/ASTUtils.java

```diff
@@ -190,4 +196,53 @@ private static String getNodeFullyQualifiedName(Optional<Node> nodeOptional) {
             return "";
         }
     }
+
+    public static Optional<JavadocComment> attemptToFindJavadocComment(BodyDeclaration<?> bodyDeclaration) {
+        // It is possible to have a situation where a Javadoc exists but is in a detached from the declaration body.
+        //
+        // This occurs in a situation such as the following:
+        //
+        // <javadoc>
+        // // Comment between Javadoc and body declaration
+        // <body declaration>
+        //
+        // Unfortunately, this will be flagged in ApiView as missing a Javadoc. So, now, traverse the comments of the
+        // body declaration until they terminate, then find the orphaned (if there is any) Javadoc which precedes it in
+        // the code file. This is only attempted if bodyDeclaration doesn't have a Javadoc.
```

Comment:
This can be the javadoc comment for this method instead of a comment inside the method body.